src/versions.rs

```diff
@@ -140,22 +286,106 @@ async fn get_plan_packages(
     Ok(plan_packages)
 }
 
+/// Represents a single branch in a repository.
+///
+/// This struct is used to deserialize JSON data from a repository's branch list.
 #[derive(serde::Deserialize, Debug)]
 struct Branch {
+    /// The name of the branch.
     name: String,
 }
 
-pub async fn get_release_branches_versions() -> Result<Vec<String>, Box<dyn std::error::Error>> {
+/// Contains information about a repository.
+///
+/// This struct holds various URLs and strings used to interact with a repository,
+/// including fetching branches and processing version information.
+struct RepositoryInfo {
+    /// The URL to fetch branch information from the repository.
+    branches_url: String,
+    /// The URL for GitHub commands related to the repository.
+    gh_cmd_url: String,
+    /// A string used to filter versions from branch names.
+    version_filter_string: String,
```

Comment:
Could possibly be a regex, but am fine with doing this in another Merge Request.  

Reason is that it does not show `stable2407` now:

![Screenshot 2024-08-14 at 13 59 16](https://github.com/user-attachments/assets/b5738a39-ac7f-4dba-8c2d-aa64841a66ca)

Reply:
Was it available earlier? I haven't changed anything such that the particular version would be skipped. I will try some regexes and maybe raise another PR for this update.

Reply:
`stable2407` is new, not sure if it showed up already. Could be that it did not, i did not check.